scripts/migration/ensure_bookmark_uniqueness.py

```diff
@@ -1,20 +1,22 @@
 import argparse
 import logging
 
-from modularodm import Q
+import django
+from django.db import transaction
 
-from framework.auth.core import User
-from framework.transactions.context import TokuTransaction
+django.setup()
+
+from osf.models import Collection, OSFUser
+from osf.modm_compat import Q
 from scripts import utils as script_utils
 from website.app import init_app
-from website.project import Node
 
 
 logger = logging.getLogger(__name__)
 
 def get_targets():
     logger.info('Acquiring targets...')
-    targets = [u for u in User.find() if Node.find(Q('is_bookmark_collection', 'eq', True) & Q('is_deleted', 'eq', False) & Q('contributors', 'eq', u._id)).count() > 1]
+    targets = [u for u in OSFUser.find() if Collection.find(Q('is_bookmark_collection', 'eq', True) & Q('is_deleted', 'eq', False) & Q('creator', 'eq', u)).count() > 1]
```

Comment:
You should be able to do something like this: `OSFUser.objects.filter(collection__is_bookmark_collection=True, collection__is_deleted=False, collection__creator_id=F('id')).aggregate(col_count=Count('collection'))`

But since it's just a migration it's probably fine.